pkg/build/controller/strategy/util.go

```diff
@@ -159,15 +158,15 @@ func setupDockerSecrets(pod *v1.Pod, container *v1.Container, pushSecret, pullSe
 	if pushSecret != nil {
 		mountSecretVolume(pod, container, pushSecret.Name, DockerPushSecretMountPath, "push")
 		container.Env = append(container.Env, []v1.EnvVar{
-			{Name: dockercfg.PushAuthType, Value: DockerPushSecretMountPath},
+			{Name: "PUSH_DOCKERCFG_PATH", Value: DockerPushSecretMountPath},
```

Comment:
don't get rid of constants.  Move them if necessary.  This is just asking for us to break stuff later.

Reply:
> don't get rid of constants. Move them if necessary. This is just asking for us to break stuff later.

Two discrete packages, which don't share a logical parent can't share constants.  This is consistent with the rest of the stack.

If you'd like to claim this is part of your API (possible), you could make the constant in the API package, but as an intermediate step (since API changes are impractical until openshift/api becomes canonical), this gets us where we need to be.